Source/Initialization/PlasmaInjector.cpp

```diff
@@ -79,14 +79,22 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
     pp.query("radially_weighted", radially_weighted);
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(radially_weighted, "ERROR: Only radially_weighted=true is supported");
 
-    // parse plasma boundaries
-    xmin = std::numeric_limits<amrex::Real>::lowest();
-    ymin = std::numeric_limits<amrex::Real>::lowest();
-    zmin = std::numeric_limits<amrex::Real>::lowest();
-
-    xmax = std::numeric_limits<amrex::Real>::max();
-    ymax = std::numeric_limits<amrex::Real>::max();
-    zmax = std::numeric_limits<amrex::Real>::max();
+    // box boundaries
+    // FIXME: Only injects particles on coarsest grid, so changes are needed to allow injection to finer grid levels.
+    const Geometry& geom = WarpX::GetInstance().Geom(0);
```

Comment:
Object lifetime question: is the `PlasmaInjector` class created once at the simulation start, which will probably return the t=0 sim box extend here, or again every time step? If the former, then we might cause problems fixing it to the t=0 geometry with continuous injection in moving windows, e.g. for brackground plasma.

Reply:
That problem would already occur without these changes, when we define `zmin` and `zmax` values for plasmas that are injected from out of the box, or not?

Reply:
Discussed offline: we will make an if that tests if moving window is being used. If that is the case then we set `zmin,zmax` to -/+inf

Reply:
We decided to set zmin,zmax to -/+inf by default and only change it if in that direction periodic boundary conditions are being used.